cpp/bench/ann/src/hnswlib/hnswlib_benchmark.cpp

```diff
@@ -116,7 +116,9 @@ REGISTER_ALGO_INSTANCE(std::int8_t);
 REGISTER_ALGO_INSTANCE(std::uint8_t);
 
 #ifdef ANN_BENCH_BUILD_MAIN
+#ifndef CPU_ONLY
 #define CPU_ONLY
+#endif
```

Comment:
What do you think about moving `CPU_ONLY` to the `ConfigureAnnBench`  function in CMakeLists.txt as an option?